.travis.yml

```diff
@@ -4,3 +4,19 @@ node_js:
   - '0.10'
   - '0.12'
   - 'iojs'
+
+cache:
+  directories:
+    - node_modules
+
+matrix:
+  fast_finish: true
```

Comment:
Why do we need `fast_finish`, we [don't have any allowed failures](http://blog.travis-ci.com/2013-11-27-fast-finishing-builds/), so then it doesn't make much sense, right?

And if we would have allowed failures, I always prefer if they don't fast finish, tbh.